### PR TITLE
Update portal ID for booster differently

### PR DESCRIPTION
**Pull request by skaex**
### Description
This allows for portal ID for booster to be updated differently on SF.

### Motivation and Context
This was required from the data team

https://app.asana.com/0/1178906757852153/1195445524446161

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [x] My code follows the code style of this project.
- [x] I have added neccessary labels
- [x] My PR title follows the convention in the guide
- [x] I have assigned myself to the PR
- [ ] I have requested reviews from team members
- [ ] I have created a task for reviewers on PM tool
- [ ] All new and existing tests passed.

### CODE REVIEW

File: app/services/setup_portal_account.rb [hunk 1]
```diff
@@ -53,6 +57,10 @@ def should_create_join_user?
     ENV['CREATE_JOIN_USER_ON_SIGN_UP'].present?
   end
 
+  def booster_instance?
+    ENV['BOOSTER_INSTANCE'].present?
```
Comment (rshipp) [inline app/services/setup_portal_account.rb:61]: should this just be `ENV['BOOSTER_INSTANCE']`? I can't find any reference to what `present?` is

Comment (skaex) [reply app/services/setup_portal_account.rb:61]: Here's one on api dock: https://apidock.com/rails/v6.0.0/Object/present%3F . From the method signature, I want a boolean indicating if this is a booster instance or not.

Comment (rshipp) [reply app/services/setup_portal_account.rb:61]: Ahh ok, thanks for the link!

**Review by sadleb — CHANGES_REQUESTED** [review]
We already have an APPLICATION_HOST ENV var. Would it be cleaner and less maintainance to use that to determine which Salesforce field to set?

**Comment by skaex** [comment]
> We already have an APPLICATION_HOST ENV var. Would it be cleaner and less maintainance to use that to determine which Salesforce field to set?

I thought of this but decided against it. My reasoning was that this would couple this implementation to the booster url

```ruby
def booster_instance?
    ENV['APPLICATION_HOST'].eql?('boosterplatform.braven.org')
end
```

Whenever the variable changes, we have to change the code too.

**Comment by sadleb** [comment]
> > We already have an APPLICATION_HOST ENV var. Would it be cleaner and less maintainance to use that to determine which Salesforce field to set?
> 
> I thought of this but decided against it. My reasoning was that this would couple this implementation to the booster url
> 
> ```ruby
> def booster_instance?
>     ENV['APPLICATION_HOST'].eql?('boosterplatform.braven.org')
> end
> ```
> 
> Whenever the variable changes, we have to change the code too.

I can’t imagine us changing it, but I’m fine with this since the plan is for both Booster and Portal to go away by next fall so it’s not worth discussing in too much more depth.

**Review by sadleb — APPROVED** [review]

**Pull request merged**